src/env.d/03-dbus

```diff
@@ -0,0 +1,9 @@
+# Dbus settings
+DBUS_SYSTEM_BUS_ADDRESS='unix:path=/run/dbus/system_bus_socket'
```

Comment:
Do you mean `unix:path=/var/run/dbus/system_bus_socket`?

Reply:
This line fixes skycocker/chromebrew#5864 as well :)

Reply:
I have both on my system... what should we do here?

Reply:
Do you have both on your system?

Reply:
I am not able to check it now, but I am sure that `/var/run/dbus/system_bus_socket` exists on my system

Reply:
> I have both on my system... what should we do here?

Might be a symlink or same device file?

Reply:
Changed it to `unix:path=/var/run/dbus/system_bus_socket` and confirmed it works with `vmc`...